src/net/relay.rs

```diff
@@ -753,14 +752,15 @@ impl Relayer {
         }
         // process as many epochs as we can.
         // Try to process at least a few epochs
-        let receipts: Vec<_> = chainstate.process_blocks(burndb, new_blocks.len() + 50)?.into_iter()
+        let max_epochs = if new_blocks.len() < 3 { 3 } else { new_blocks.len() };
```

Comment:
Why set a minimum to three?

Reply:
It needs to be at least 1 in order to process forks that we didn't get blocks for.  It was originally 50.  It won't really matter because it won't do more work than needed (the value passed as `max_epochs` is the upper bound on the number of attachable blocks to consider).